examples/hellocluster-yaml/launch_hellocluster.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import yaml
+import logging
+
+from slurmpilot.config import default_cluster_and_partition
+from slurmpilot.slurm_wrapper import SlurmWrapper, JobCreationInfo
+from slurmpilot.util import unify
+
+def load_config(config_path):
+    """
+    Load configuration from a YAML file.
+    :param config_path: Path to the YAML configuration file.
+    :return: Parsed configuration as a dictionary.
+    """
+    with open(config_path, 'r') as file:
+        config = yaml.safe_load(file)
+    return config
+
+def main():
+    parser = argparse.ArgumentParser(description="Load configurations from a YAML file.")
```

Comment:
Is there a reason for not using omegaconf if we use YAML to configure CLI?

Reply:
I've used OmegaConf in the latest commit :)